### PR TITLE
M0127CC-628 Move SDK license usage tutorial

### PR DESCRIPTION
**Pull request by fpinel-microej**

### CODE REVIEW

File: Tutorials/tutorialLicense.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+---------------
+
+#. If you did not received an activation certificate, please contact the support@microej.com.
+#. Check that you have your Activation Key.
+#. Check that your dongle is activated.
+#. In the SDK, in :guilabel:`Window` > :guilabel:`Preferences` > :guilabel:`MicroEJ`,
+
+    #. Check that the dongle appears on the ``Licenses`` list.
+
+        - If not, check that the dongle is plugged in your computer. If using Linux `please configure the USB rules <https://forum.microej.com/t/how-can-i-use-a-microej-usb-dongle-on-linux/223/2>`_.
```
Comment (fpinel-microej) [inline Tutorials/tutorialLicense.rst:103]: Should this forum post be converted to a docs page instead of linking to it ?

**Review by fpinel-microej — COMMENTED** [review]
Here is the original post converted to RST, please on if and how would we like to marge it with https://docs.microej.com/en/latest/ApplicationDeveloperGuide/licenses.html

File: Tutorials/tutorialLicense.rst (hunk 1 above)
Comment (stephanemainchain) [reply Tutorials/tutorialLicense.rst:103]: yes, we can add a few lines and avoid linking to the forum

**Comment by stephanemainchain** [comment]
> 
> 
> Here is the original post converted to RST, please advise on if and how would we like to marge it with https://docs.microej.com/en/latest/ApplicationDeveloperGuide/licenses.html

yes we could either merge it here in Application dev or in the overview section (personnally overview is better). @frivieremicroej ?

**Comment by stephanemainchain** [comment]
Add a link in the intro for evaluation license:

MicroEJ provides two types of licenses :

    Evaluation: can be found on https://repository.microej.com/architectures/
    Production: can be requested to MicroEJ support team support@microej.com

**Comment by fpinel-microej** [comment]
Merged the new tutorial page with the original license page, I will delete the page from the tutorials once there's no issues and we're done with the merging

**Comment by frivieremicroej** [comment]
> > Here is the original post converted to RST, please advise on if and how would we like to marge it with https://docs.microej.com/en/latest/ApplicationDeveloperGuide/licenses.html
> 
> yes we could either merge it here in Application dev or in the overview section (personnally overview is better). @frivieremicroej ?

yes I moved to overview

**Pull request merged**